src/components/parent-form/ParticipantInfoPage.tsx

```diff
@@ -51,110 +64,96 @@ export const ParticipantInfoPage: React.FC<ParticipantPageProps> = ({
                     be provided afterwards.
                 </Text>
             </Box>
-            <FormLabel>
-                Participant Name
-                <HStack spacing="24px">
-                    <FormControl id="participant-first-name">
-                        <Input
-                            placeholder="First name"
-                            onChange={(e) =>
-                                props.setParticipantFirstName(e.target.value)
-                            }
-                            value={props.participantFirstName}
-                        />
-                    </FormControl>
-                    <FormControl id="participant-last-name">
-                        <Input
-                            placeholder="Last name"
-                            onChange={(e) =>
-                                props.setParticipantLastName(e.target.value)
-                            }
-                            value={props.participantLastName}
-                        />
-                    </FormControl>
+            <>
+                <HStack spacing="24px" style={{ height: "100px" }}>
+                    <TextField
+                        name="First Name"
+                        value={props.participantFirstName}
+                        setValue={props.setParticipantFirstName}
+                    ></TextField>
+                    <TextField
+                        name="Last Name"
+                        value={props.participantLastName}
+                        setValue={props.setParticipantLastName}
+                    ></TextField>
                 </HStack>
-            </FormLabel>
-            <FormControl id="date-of-birth">
-                <FormLabel>Date Of Birth (YYYY-MM-DD) </FormLabel>
-                <Input
-                    placeholder="Date Of Birth"
-                    onChange={(e) => props.setDateOfBirth(e.target.value)}
+                <DateField
+                    name={"Date Of Birth"}
                     value={props.dateOfBirth}
-                />
-            </FormControl>
-            <FormControl id="street-address-1">
-                <FormLabel>Street Address 1</FormLabel>
-                <Input
-                    placeholder="815 Hornby St."
-                    onChange={(e) => props.setAddress1(e.target.value)}
+                    setValue={props.setDateOfBirth}
+                ></DateField>
```

Comment:
I think the `DateField` comp can be self-closing
```suggestion
                />
```